code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -159,8 +159,9 @@
 
 	var/area/vent_area = get_area(src)
 	if(!GLOB.air_vent_names[id_tag])
-		// If we do not have a name, assign one
-		name = "[assign_random_name()] [vent_area.name] Vent Pump" // matching case
+		// If we do not have a name, assign one.
+		// Produces names like "Port Quarter Solar vent pump hZ2l6".
+		name = "\proper [vent_area.name] vent pump [assign_random_name()]"
```

Comment:
```suggestion
		name = "\proper [vent_area.name] vent pump #[assign_random_name()]"
```
thoughts?

Reply:
It's not a number though.

Reply:
I did consider `#` after I had already put the PR up and then spent a little more time staring at it. It seems fine either way.